src/Workspaces/CSharp/Portable/Simplification/CSharpSimplificationService.cs

```diff
@@ -31,7 +28,8 @@ internal partial class CSharpSimplificationService : AbstractSimplificationServi
                 new CSharpParenthesesReducer(),
                 new CSharpEscapingReducer(),
                 new CSharpMiscellaneousReducer(),
-                new CSharpInferredMemberNameReducer());
+                new CSharpInferredMemberNameReducer(),
+                new CSharpDefaultExpressionReducer());
```

Comment:
If users have their code style set to prefer `default` instead of `default(T)`, the simplifier will now perform the substitution where applicable.